test/AutoRest.TestServer.Tests/additionalProperties.cs

```diff
@@ -130,6 +131,7 @@ public class AdditionalPropertiesTest : TestServerTestBase
         });
 
         [Test]
+        [IgnoreOnTestServer(TestServerVersion.V2, "No match")]
```

Comment:
Were these tests removed from V2?

Reply:
We are sending different format now that is correct but V2 match is too strict.